TerrariaServerAPI/TerrariaServerAPI.csproj

```diff
@@ -8,7 +8,7 @@
 		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 		<GenerateDocumentationFile>True</GenerateDocumentationFile>
 		<RunPostBuildEvent>Always</RunPostBuildEvent>
-		<Version>5.0.0-beta.0.0</Version>
+		<Version>5.0.0-beta.0.1</Version>
```

Comment:
Hm I wonder if we can get it to tick to 5.0.1 instead @hakusaro ?
No rush, can do it manually when back at a pc if you prefer for now?

Reply:
I'm fine with this actually